website/homepage/src/docs/event_template.html

```diff
@@ -0,0 +1,60 @@
+{% for obj in event["fns"] %}
```

Comment:
This is basically obj_doc_template but with all the component-specific logic removed and with some minor 

I decided to add a new template for events because events are different from our other apis. Users are not intended to use the `Clickable` class directly, instead they use the `click` method on the associated component. I felt the existing template did not accommodate this difference.

So what this template does is treat the methods of each Event class as if they were a top-level function.